src/app/settings/action-triggers/components/action-triggers-list/action-triggers-list.component.html

```diff
@@ -9,7 +9,7 @@
         (onSelect)="applyUserActionToSelectedActionTriggers($event)"
         label="Selected">
       </sofie-dropdown-button>
-      <sofie-button (onClick)="setSelectMode(false)" label="Disable multiselect"></sofie-button>
+      <sofie-button (onClick)="setSelectMode(false)" label="Disable multiselect" [buttonClasses]="'c-sofie-button c-sofie-button-primary ms-2'"></sofie-button>
```

Comment:
1. When we add class we add this class to native angular elements not to the button witch will be content of the container. I Want to apply it on <button> not on <sofie-button>. We can change the scss selectors and work with class applied to <sofie-button>. In reality the different is verry small
2 and 3. If it's used only there I will move in sofie-button scss

Reply:
@LindvedKrvang 2 and 3 are done

Reply:
> I Want to apply it on not on

I don't follow what you mean here.

1 & 2: You are still parsing the CSS class from the calling component to the SofieButton component each and every time you use the SofieButton. There is currently not a single usage of SofieButton where `c-sofie-button` isn't also been given to it. Looking at the CSS for `c-sofie-button` it looks like that this class is quite essential for the button to look correctly. If that's true, then we shouldn't need to remember to give the component this class every time we use it, it should simply always have the class.

So instead of writing: `<sofie-button buttonClasses="c-sofie-button" ... ></sofie-button>` you should just write `<sofie-button ... ></sofie-button>`

Speaking of `buttonClasses`. This attribute needs to be removed. It's used to parse CSS classes from the outside of the component to the button component (which we established above we shouldn't). In the Dialog component you are parsing `buttonClasses` throught _at least three_ components, just to give SofieButton a CSS class that it should already be setting itself.

3. You are still parsing it as a CSS class instead of a type exposed by on SofieButton:
```
# sofie-button.ts
@input() public type: 'PRIMARY' | 'WARNING' | 'DANGER' = 'PRIMARY'

#some component that wants a primary button
<sofie-button></sofie-button>

#some component that wants a warning button
<sofie-button type="WARNING"></sofie-button>

#some component that wants a danger button
<sofie-button type="DANGER"></sofie-button>

```
Above solution is typesafe and all calling components don't need to know an intimate detail of SofieButton (the CSS class). Instead they can just see the exposed types available for the button.

-----

Formulating the above answer made me take a closer look at the implementation of `SofieButton` which lead me to some comments about the SofieButton: 

The attribute `onClick: EventEmitter` is redundant and should be removed. All it does is letting us write `<sofie-button (onClick)="..."></sofie-button>`. On HTML elements there is already a `(click)` event and we should just use that. So it becomes: `<sofie-button (click)="..." ></sofie-button>`.

The attribute `label` should ideally be removed. Right now, SofieButton can only ever be a "text" button. Instead of having a label attribute, we should just content inject whatever needs to be in the button:
So instead of writing `<sofie-button label="Click me!"></sofie-button>` we should write `<sofie-button>Click me!</sofie-button>`. This makes the component a lot more flexible.

The attribute `buttonClasses` needs to be removed as discussed further above.

The attribute `type` needs to be more typesafe. Right now it is `public type: string = 'button'`. You use it to indicate if it is a submit button or not, so it needs to atleast be: `public type: 'button' | 'submit' = 'button'`. 
I would however make the argument that this attribute also should be deleted.